cve.md

```diff
@@ -0,0 +1,7 @@
+# CVE Protocols
+
+Kubecost will never ask for or access your confidential data, and we try to ensure your information is always secure. Here is our proactive approach for ensuring common vulnerabilities and exposures (CVEs) are spotted and handled to ensure our product remains safe to use.
+
+Due to Kubecost’s multiple base dependencies, it’s important to track vulnerabilities in our product as well as those dependencies. We use [Aqua’s Trivy](https://github.com/aquasecurity/trivy) to scan our containers as well as any [Helm chart](https://github.com/kubecost/integration-ci-cd/blob/main/.github/workflows/nightly-scan-dependencies.yaml) dependencies nightly to track potential CVEs. The specific action used can be viewed [here](https://github.com/aquasecurity/trivy-action). Potential vulnerabilities will always be fixed in our upcoming release.
```

Comment:
The `[Helm chart]` link won't work for non-Kubecost folks because kubecost/integration-ci-cd is private.